core/core_bind.cpp

```diff
@@ -68,6 +67,18 @@ Ref<Resource> ResourceLoader::load_threaded_get(const String &p_path) {
 	Ref<Resource> res = ::ResourceLoader::load_threaded_get(p_path, &error);
 	return res;
 }
+PackedStringArray ResourceLoader::get_cached_paths() {
```

Comment:
```suggestion

PackedStringArray ResourceLoader::get_cached_paths() {
```
For all you added, please leave an empty line between methods in the `cpp` files